Update README.md (#5410)
SkriptLang/Skript: README.md

before reporting issues.

## Help Us Test
Wanting to help test Skript's new features and releases?

> You can head on over to our [Official Testing Discord](https://discord.gg/ZPsZAg6yg), and whenever we start testing new features/releases you will be the first to know.

Reviewer: Link is broken, you're missing a `u` at the end `ZPsZAg6ygu`

Author: Thanks for catching that

> Please note this is not a HELP discord!
> If you require assistance with how to use Skript please check out the [Relevant Links](https://github.com/SkriptLang/Skript#relevant-links) section for a list of available discords/docs to assist you.

Reviewer 2: ```suggestion
Please note this is not a help Discord.
If you require assistance with how to use Skript please check out the [Relevant Links](https://github.com/SkriptLang/Skript#relevant-links) section for a list of available resources to assist you.
```

## Relevant Links
* [skUnity forums](https://forums.skunity.com)

> * [skUnity Add-on releases](https://forums.skunity.com/forums/addon-releases)

Reviewer 2: ```suggestion
* [skUnity addon releases](https://forums.skunity.com/forums/addon-releases)
```

> * [skUnity Discord invite](https://discord.gg/skript)

Reviewer 2: ```suggestion
* [skUnity Discord invite](https://discord.gg/0l3WlzBPKX7WNjkf)
```
The invite link you mentioned doesn't have the most stable track record

Reviewer: But `.gg/skript` is the custom one which is better and more rememberable

Reviewer 2: I talked to Kenzie about it, skUnity had lost the URL before so I don't think we should rely on it

## Relevant Links
* [skUnity forums](https://forums.skunity.com)

> * [skUnity Documentation](https://docs.skunity.com)

Reviewer 3: You should link to the official Skript docs instead.

Reviewer: Skript official docs already have its own section above, so the skUnity docs should be just removed

Author: While I it's still link it might be good to add it there as well or see if there's anymore information that could be added to the section, It's basically a one liner and just like help us test isn't noticeable
